### ISSUE TITLE
[DataCap Application] <Human PanGenomics Project> - <HPGP-1>

### ISSUE THREAD

**Issue by helloworld09911** [pandacrypto/DSPA-Allocator#14]
### Version

1

### DataCap Applicant

Neal

### Project ID

14

### Data Owner Name

National Human Genome Research Instititue

### Data Owner Country/Region

United States

### Data Owner Industry

Life Science / Healthcare

### Website

https://humanpangenome.org/

### Social Media Handle

https://humanpangenome.org/

### Social Media Type

Other

### What is your role related to the dataset

Data Preparer

### Total amount of DataCap being requested

5PiB

### Expected size of single dataset (one copy)

580TiB

### Number of replicas to store

5

### Weekly allocation of DataCap requested

1000TiB

### On-chain address for first allocation

f1uzqjbchigrmafszkfxl2tpkyu2amp2z2uwuhuqq

### Data Type of Application

Public, Open Dataset (Research/Non-Profit)

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
Personal data preparer with confidence in the future of filecoin--The Human Pangenome Reference Consortium (HPRC) is a project funded by the National Human Genome Research Institute to sequence and assemble genomes from individuals from diverse populations in order to better represent genomic landscape of diverse human populations.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
This dataset includes sequencing data, assemblies, and analyses for the offspring of ten parent-offspring trios.

The raw data is about 1.2P and will transfer to 2P in raw power per copy considering the padding.  I will distribute 10x copies to different SP in various geo.

Consider storage plan compliance and apply for 5PiB for trial first.Work with 5 SPs first.
```

### Where was the data currently stored in this dataset sourced from

AWS Cloud

### If you are a data preparer. What is your location (Country/Region)

None

### Please share a sample of the data

```text
https://github.com/human-pangenomics/hpgp-data
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Sporadic

### For how long do you plan to keep this dataset stored on Filecoin

1.5 to 2 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, North America, South America, Europe, Australia (continent)

### How will you be distributing your data to storage providers

HTTP or FTP server, Shipping hard drives

### How did you find your storage providers

Slack, Partners

### Please list the provider IDs and location of the storage providers you will be working with.

```text
f02953066
f02956073 
f02837684 
f03028412 
f02956073
f03028412
```

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by clriesco** [comment]
There was a problem submitting your application. A third party component is down for the moment. We will trigger this manually as soon as it's up and running again.

**Comment by helloworld09911** [comment]
> There was a problem submitting your application. A third party component is down for the moment. We will trigger this manually as soon as it's up and running again.

Thanks for the prompt feedback, looking forward to a quick resolution, cheers!

**Comment by helloworld09911** [comment]
Hi @clriesco,any updation?

**Comment by helloworld09911** [comment]
Hi @clriesco Can I get some help?Thanks！～

**Comment by clriesco** [comment]
Done

**Comment by helloworld09911** [comment]
Yes, SPs themselves are stocked with enough pledged coins.

Individual data preparers, mainly with the help of official tools such as boost, lotus, Singularity. the dataset used to be stored on filecoin, but the volume is not very large, and most of the SPs communicating at the same time have not stored this dataset.

Currently, we are considering communicating with SPs such as f02953066, f02956073, f02837684, f03028412, f02814775, f02814770.

**Comment by helloworld09911** [comment]
The following SPs working together：
f03080854  ZT  US       
f02956073  Alice  SG
f02815438 MaiJie India
f03080852  Yming  Canada  
f02227496 Douyun  HK

**Comment by pandacrypto** [comment]
Please send the identity information of node f02815438 to official@dspa-asia.net for verification.thanks！

**Comment by pandacrypto** [comment]
checker:manualTrigger

**Comment by helloworld09911** [comment]
Hello@pandacrypto I asked SPs and they told me that there were some bugs before but they are fixed now, you can check again.

**Comment by pandacrypto** [comment]
checker:manualTrigger

**Comment by pandacrypto** [comment]
Still has bugs @helloworld09911

**Comment by pandacrypto** [comment]
checker:manualTrigger

**Comment by pandacrypto** [comment]
The report shows that the SP retrieval rate is still low, please contact SPs to continuously adjust the configuration to meet the Spark retrieval requirements.

**Comment by pandacrypto** [comment]
checker:manualTrigger

**Comment by pandacrypto** [comment]
The SPs involved in the actual sealing are consistent with the disclosed information.
For further KYC, please send your identification information to official@dspa-asia.net. The email should include the issue number.
Thanks！

**Comment by helloworld09911** [comment]
The email has been sent. Please check your inbox. Thank you.

<img width="626" alt="image" src="https://github.com/user-attachments/assets/ecb32d6e-d259-4e6c-9362-9c061aa51cdb">

**Comment by pandacrypto** [comment]
Email received. Please continue to maintain your efforts!

**Comment by helloworld09911** [comment]
Promptly disclose new SPs information.
f03173124 Aomeige South Korea
f03188443  YPH  SG

**Comment by helloworld09911** [comment]
Continuing with the advance disclosure, new sealing will be initiated within one week.Also communicating with these SPs (f03196401, f03196399).
Laibin - Dubai (f03190614)
Fei - Vancouver, Canada (f03188440)
YM - Japan (f03190616)

**Comment by pandacrypto** [comment]
checker:manualTrigger

**Comment by pandacrypto** [comment]
Overall, it looks good, but there is still the issue that "⚠️ 40.00% of Storage Providers have a retrieval success rate of less than 75%." We will proceed with a limited Datacap allocation of 1 PiB. Please confirm that you will support more compliant SPs. If future reports show continued improvement, the allocation limit will be increased to 2 PiB.

**Comment by helloworld09911** [comment]
Yes, we are working hard and hope to secure 2 PiB of Datacap next time.Thanks！

**Comment by pandacrypto** [comment]
checker:manualTrigger

**Comment by filecoin-watchdog** [comment]
checker:manualTrigger

**Comment by pandacrypto** [comment]
checker:manualTrigger

**Comment by kacperzuk-neti** [comment]
checker:manualTrigger

**Comment by helloworld09911** [comment]
checker:manualTrigger

**Comment by helloworld09911** [comment]
Updated, about to start a new sealing:
f03201941 YanLong Canada
f03228953 K cloud Thailand

**Comment by pandacrypto** [comment]
> Overall, it looks good, but there is still the issue that "⚠️ 40.00% of Storage Providers have a retrieval success rate of less than 75%." We will proceed with a limited Datacap allocation of 1 PiB. Please confirm that you will support more compliant SPs. If future reports show continued improvement, the allocation limit will be increased to 2 PiB.

The client communicated with me via slack and stated that they will not be working with DDO clients for a while and that Spark retrieval rates will continue to improve. Willing to support this round and reinstate 2PiB Datacap approval based on good track record.

**Comment by helloworld09911** [comment]
We are scheduled to initiate the new sealing within the upcoming week.
f03251828 Dallas, USA 
f03251827 Singapore 
f03254061 Canada 
f03254063 New York, USA 
f03254125 Dallas, USA 
f03259560 Japan

**Comment by pandacrypto** [comment]
checker:manualTrigger

**Comment by AlanGreaterheat** [comment]
checker:manualTrigger

**Comment by pandacrypto** [comment]
checker:manualTrigger

**Comment by pandacrypto** [comment]
Please explain why the proportion of SPs nodes retrieving as 0 in Spark has increased?

**Comment by helloworld09911** [comment]
Thank you for your patient review @pandacrypto. The Spark retrieval rate of 0% has not actually shown significant improvement; previously, the SPs that used DDO were displayed as '—' in Spark, and now they are shown as '0'. We suspect that this change is due to adjustments made by the Spark team.

In earlier times, because the Spark community did not support DDO, the SPs that originally used DDO had long ceased cooperation.Please continue to support us.

**Comment by helloworld09911** [comment]
Latest Report：
<img width="1356" alt="Image" src="https://github.com/user-attachments/assets/2317f666-d157-4504-b180-07902443bef3" />

——————

Old Report：
<img width="1159" alt="Image" src="https://github.com/user-attachments/assets/d1d9f8f7-c3b7-48f1-9b80-77f26d486f78" />
<img width="1117" alt="Image" src="https://github.com/user-attachments/assets/ab18aab3-ace7-4d05-b909-cf72398410a9" />

**Comment by pandacrypto** [comment]
Due to a total quota application of 5 PB, support is provided for the final round of 0.5PiB
